spring-cloud-stream-app-dependencies/pom.xml

```diff
@@ -654,8 +660,11 @@
 				<artifactId>pmml-evaluator</artifactId>
 				<version>${pmml.version}</version>
 			</dependency>
-
-
+			<dependency>
+				<groupId>javax.mail</groupId>
+				<artifactId>mail</artifactId>
+				<version>1.5.0-b01</version>
```

Comment:
We don't need this dependency because Boot provides for us:
```
<dependency>
	<groupId>com.sun.mail</groupId>
	<artifactId>javax.mail</artifactId>
</dependency>
```
with the version of `1.5.5`.
That is the reason do not use `javax.mail:mail` anymore, because this one hasn't been updated since March 2013.